src/components/Card/Card.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import styled from "styled-components";
+import { Composition, Box } from "atomic-layout";
+import CardOptions from "../CardOptions";
+
+interface CardProps {
+  imageUrl: string;
+  title: string;
+  genre: string;
+  releaseDate: string;
+}
+
+const CardStyle = styled.div`
+  height: 33rem;
+  color: ${(props) => props.theme.colour.grey};
+`;
+
+const Image = styled.img`
```

Comment:
It's a good practice to call such components like "StyledImage" so that you'll be able to separate them visually from the regular components.
Also it's a good practice to move such components to "styled" folder inside the main component folder. So that your regular components become more clear.